pkg/plugins/builtin/kafka/kafka.go

```diff
@@ -0,0 +1,174 @@
+package kafka
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/Shopify/sarama"
+
+	"github.com/ovh/utask/pkg/plugins/taskplugin"
+)
+
+// the Kafka plugin produces a Kafka message
+var (
+	Plugin = taskplugin.New("kafka", "1.0", exec,
+		taskplugin.WithConfig(validConfig, KafkaConfig{}),
+		taskplugin.WithResources(resourcesKafka),
+	)
+)
+
+const (
+	// TimeoutDefault represents the default value that will be used for the request, if not defined in configuration
+	TimeoutDefault    = "10s"
+	DefaultMaxRetries = 5
+)
+
+type Message struct {
+	Topic string `json:"topic"`
+	Key   string `json:"key,omitempty"`
+	Value string `json:"value"`
+}
+
+// KafkaConfig is the configuration needed to write a message on Kafka topic
+type KafkaConfig struct {
+	Brokers      []string `json:"brokers"`
```

Comment:
In order to allow templatisation of this field, e.g
```
{{ field `config` `kafka-config` `brokers`}}`)
```
This better be a string, comma-separated.